projectroles/context_processors.py

```diff
@@ -48,3 +56,38 @@ def app_alerts_processor(request):
                 .count()
             }
     return {'app_alerts': 0}
+
+
+def sidebar_processor(request):
+    """
+    Context processor for providing sidebar information.
+    """
+
+    def get_sidebar_icon_size():
+        """Return sidebar icon size with a min/max limit"""
+        return sorted(
+            [
+                SIDEBAR_ICON_MIN_SIZE,
+                getattr(settings, 'PROJECTROLES_SIDEBAR_ICON_SIZE', 32),
+                SIDEBAR_ICON_MAX_SIZE,
+            ]
+        )[1]
+
+    def get_sidebar_notch_pos():
```

Comment:
We don't need to have separate functions for these, we can simply return them in the return value dict instead. Maybe the `sidebar_icon_size` we should populate in a separate variable first to avoid ugly nesting in code.